curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md

```diff
@@ -36,35 +36,35 @@ tests:
 ```html
 <body>
   <header>
-    <h1>Training</h1>
+    <h1>Entrenamiento</h1>
     <nav>
       <ul>
-        <li><a href="#stealth">Stealth &amp; Agility</a></li>
-        <li><a href="#combat">Combat</a></li>
-        <li><a href="#weapons">Weapons</a></li>
+        <li><a href="#stealth">Sigilio & Agilidad</a></li>
```

Comment:
You mean **sigilo** ?

Reply:
Thanks! Not sure what happened there. I've corrected it.